### ISSUE TITLE
"Deployment not found" on all deployments

### ISSUE THREAD

**Issue by holograph** [aptakube/aptakube#44]
### Current and expected behavior

I was just starting a trial of Aptakube. It connected to the cluster (EKS if it matters), showed the relevant namespaces and deployments, but clicking on any deployment shows a message as follows:

>:warning:
>Deployment *elided* not found in *elided* namespace
>This deployment does not exist or may have been deleted.
><elided, URL-encoded cluster ARN>

kubectl, Octant and OpenLens (bless its soul) all function correctly.

### Operating System

MacOS 13.0.1 (22A400)

### Environment

```
clientVersion:
  buildDate: "2022-12-08T19:51:43Z"
  compiler: gc
  gitCommit: b46a3f887ca979b1a5d14fd39cb1af43e7e5d12d
  gitTreeState: clean
  gitVersion: v1.26.0
  goVersion: go1.19.4
  major: "1"
  minor: "26"
  platform: darwin/arm64
kustomizeVersion: v4.5.7
serverVersion:
  buildDate: "2022-10-24T20:35:40Z"
  compiler: gc
  gitCommit: 55bd5d5cb7d32bc35e4e050f536181196fb8c6f7
  gitTreeState: clean
  gitVersion: v1.23.13-eks-fb459a0
  goVersion: go1.17.13
  major: "1"
  minor: 23+
  platform: linux/amd64
  ```

**Comment by goenning** [comment]
Does your context name have @ or another special char? If yes, it might be the same as this bug https://github.com/aptakube/aptakube/issues/42

Can you try renaming the context (even if temporarily) just to confirm if it's the same bug? 

Bug fix coming out tomorrow.

**Comment by goenning** [comment]
Hey Tomer, the bug I mentioned above is fixed on 1.0.1, can you give it a try? The app will prompt for auto-update within 5 seconds after opening it.

**Comment by holograph** [comment]
Seems to work with 1.0.2, thanks!
